source/includes/_dataset_status.md

```diff
@@ -2,6 +2,25 @@
 
 ## The dataset status object
 
+> Example object
+
+```json
+{
+  "name": "idle",
+  "previous": "processing",
+  "next": null,
+  "since": "2018-01-21T11:16:08.017412+00:00",
+  "published": true,
+  "records_errors": [],
+  "dataset": {
+    "dataset_uid": "da_7jgvnj",
+  }
+}
+```
+
+<aside class="notice">
+This API needs to be refactored into something clearer.
+</aside>
```

Comment:
I think this should not be in the documentation but rather on a todo list somewhere :)

Reply:
I thought it was a good idea to let people know in advance that we are going to break this entry point in the future
but no worries, I'll remove the notice